i18ng.js

```diff
@@ -64,7 +88,7 @@ angular.module('i18ng')
 
         var translations = {}
         var t = translate.bind(null, scope, element, translations)
-        var ignore = ['opts', 'html']
+        var ignore = ['opts', 'html', 'nestedhtml']
```

Comment:
ah ok, so `i18ng-tag-name` will only go on elements in the content of the tag being parsed, and that content would have been wiped away by this directive anyway. Got it